pkg/okteto/client_test.go

```diff
@@ -46,6 +46,9 @@ func (fc fakeGraphQLClient) Query(ctx context.Context, q interface{}, _ map[stri
 }
 func (fc fakeGraphQLClient) Mutate(ctx context.Context, m interface{}, _ map[string]interface{}) error {
 	if fc.mutationResult != nil {
+		if isAnInterface(m) {
```

Comment:
genuine question, why do we need this check?

Reply:
This is to get the struct that implements the interface. Just for mocking the behavior of gql call